src/Datadog.Trace/Util/DelegatesHelper.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Diagnostics;
+using System.Reflection;
+
+namespace Datadog.Trace.Util
+{
+    /// <summary>
+    /// Delegates helper class
+    /// </summary>
+    internal static class DelegatesHelper
+    {
+        /// <summary>
+        /// Tries to get the internal ProcessExit delegate from the event using reflection.
+        /// </summary>
+        /// <returns>MulticastDelegate instance or null</returns>
+        public static MulticastDelegate GetInternalProcessExitDelegate()
+        {
+            // This methods tries to get the internal delegte for the process exit event.
+
+#if NETSTANDARD2_0 || NETCOREAPP || NETCOREAPP3_1_OR_GREATER
```

Comment:
Do we need `NETCOREAPP3_1_OR_GREATER`? Isn't `NETCOREAPP` enough?

Reply:
Do you know if NETCOREAPP includes NET5? That was the reason to add it.

Reply:
Good point. I always think of .NET Core as including .NET 5. We should check.

Reply:
Yep, .NET 5 defines `NETCOREAPP`: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives (tested and confirmed)

Reply:
Great!!! Then I will remove the “greater” one

Reply:
done in d39e660